### PR TITLE
Add readmes for UNet/ICNet trained on CamVid

### PR DESCRIPTION
**Pull request by vshampor**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
@yssemaev I'd like to remove fp32 from model name. Name icnet-fp32-onnx-0001 should be icnet-onnx-0001 because it can be as fp32 and as fp16 IR in model package

**Comment by yssemaev** [comment]
BTW, if "icnet-fp32-onnx-0001" is the original model, why not naming it like others public models, for instance, ssd512, ssd300?

**Comment by vladimir-dudnik** [comment]
@yssemaev this model distributed in IR form, so naming convention follow to other such models

**Comment by vladimir-dudnik** [comment]
@vshampor please note, there is merge conflict

File: models/intel/index.md [hunk 1]
```diff
@@ -96,7 +96,8 @@ detect areas with complex shape (for example, free space on the road).
 |------------------------------------------------------------------------------------------------------------------------------------------------|--------------------- |----------- |
 | [road-segmentation-adas-0001](./road-segmentation-adas-0001/description/road-segmentation-adas-0001.md)                                        | 4.770                | 0.184      |
 | [semantic-segmentation-adas-0001](./semantic-segmentation-adas-0001/description/semantic-segmentation-adas-0001.md)                            | 58.572               | 6.686      |
-
+| [icnet-onnx-0001](./icnet-onnx-0001/description/icnet-onnx-0001.md)                            | -               | -      |
+| [unet-onnx-0001](./unet-onnx-0001/description/unet-onnx-0001.md)                            | -              | -      |
```
Comment (IRDonch) [inline models/intel/index.md]: Why is there no complexity/size listed?

Comment (vladimir-dudnik) [reply models/intel/index.md]: there will be no info on complexity/size at the moment as model analyzer not ready yet.

Comment (vladimir-dudnik) [reply models/intel/index.md]: @vshampor is not model name is icnet-vamvid-onnx-0001 and unet-camvid-onnx-0001 ?

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Pull request merged**